cms/settings/base.py

```diff
@@ -367,7 +367,6 @@
 WAGTAIL_CONTENT_LANGUAGES = LANGUAGES = [
     ("en-gb", _("English")),
     ("cy", _("Welsh")),
-    ("uk", _("Ukrainian")),
```

Comment:
This change was made because we don't actually support a locale for Ukrainian currently, and this config now needs to align with supported locales.